test/Core.Test/AdminConsole/OrganizationFeatures/Policies/PolicyRequirements/OrganizationDataOwnershipPolicyRequirementFactoryTests.cs

```diff
@@ -30,24 +31,85 @@ public void State_WithOrganizationDataOwnershipPolicies_ReturnsRestricted(
     }
 
     [Theory, BitAutoData]
-    public void RequiresDefaultCollection_WithNoPolicies_ReturnsFalse(
-        Guid organizationId,
+    public void PolicyType_ReturnsOrganizationDataOwnership(SutProvider<OrganizationDataOwnershipPolicyRequirementFactory> sutProvider)
+    {
+        Assert.Equal(PolicyType.OrganizationDataOwnership, sutProvider.Sut.PolicyType);
+    }
+
+    [Theory, BitAutoData]
+    public void GetDefaultCollectionRequest_WithConfirmedUser(
```

Comment:
Test names should specify what is the expected outcome. In this case something like `GetDefaultCollectionRequest_WithConfirmedUser_ReturnsTrue`